File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/service/message/reactiveobject/ReadObjectAttributeRequest.java
package org.opcfoundation.webserver.service.message.reactiveobject;

import org.opcfoundation.webserver.types.common.UaObjectId;

public class ReadObjectAttributeRequest {
    private final UaObjectId objectId;

    public ReadObjectAttributeRequest(UaObjectId objectId)
    {
        this.objectId = objectId;
    }

    public UaObjectId getObjectId() {
        return objectId;
    }
}

File: opcua-webservice-node/source/src/server/service/message/reactiveobject/ReadObjectAttributeRequest.ts
import { UaObjectId } from "../../../types";

export class ReadObjectAttributeRequest {
    private readonly _objectId: UaObjectId;
    private readonly _attributeIds: Set<number>;

    constructor(objectId: UaObjectId, attributeIds: Set<number>) {
        this._objectId = objectId;
        this._attributeIds = attributeIds;
    }

    get objectId(): UaObjectId {
        return this._objectId;
    }

    get attributeIds(): Set<number> {
        return this._attributeIds;
    }
}